CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release Notes
 
 ## Platform Version 0.9.21 - UNRELEASED
+* Add a `size` field in the `fluvio partition list` output. This field represents the size of logs in the partition. ([#2214](https://github.com/infinyon/fluvio/pull/2214))
```

Comment:
Please point to the #2148

Reply:
Should I point to both the issue and the pull request?